File: app/src/main/java/com/example/meetingapp/firebase/MessagingService.java
package com.example.meetingapp.firebase;
import android.util.Log;

import androidx.annotation.NonNull;
import com.google.firebase.messaging.FirebaseMessagingService;
import com.google.firebase.messaging.RemoteMessage;


public class MessagingService extends FirebaseMessagingService {

    @Override
    public void onNewToken(@NonNull String token) {
        super.onNewToken(token);
        Log.d("BBB", "Token: "+token);
    }

    @Override
    public void onMessageReceived(@NonNull RemoteMessage remoteMessage) {
        super.onMessageReceived(remoteMessage);
        if (remoteMessage.getNotification()!=null){
            Log.d("BBB","Remote message received: "+remoteMessage.getNotification().getBody());

        }
    }
}

File: app/src/main/java/com/example/meetingapp/service/MessagingService.kt
package com.example.meetingapp.service


import android.util.Log
import android.widget.Toast
import com.example.meetingapp.R
import com.google.android.gms.tasks.OnCompleteListener
import com.google.firebase.messaging.FirebaseMessaging
import com.google.firebase.messaging.FirebaseMessagingService
import com.google.firebase.messaging.RemoteMessage


class MessagingService() : FirebaseMessagingService() {
    override fun onNewToken(p0: String) {
        super.onNewToken(p0)
        Log.d("FCM","Token : $p0")
    }

    override fun onMessageReceived(p0: RemoteMessage) {
        super.onMessageReceived(p0)
        Log.d("FCM","MessageReceived : ${p0.notification?.body}")
    }

}